manifests/init.pp

```diff
@@ -0,0 +1,22 @@
+# @summary A short summary of the purpose of this class
+#
+# A description of what this class does
+#
+# @example
+#   include pulpcore
+class pulpcore {
+
+  include selinux
+
+  class { selinux:
+    mode => 'permissive',
+  }
+
+  include ::redis
+
+  class { '::redis':
+    bind => '127.0.0.1',
+    manage_repo => true,
```

Comment:
We will want to set this to false, and configure to use the Redis in the SCL eventually, for now EPEL is fine:

http://mirror.centos.org/centos/7/sclo/x86_64/rh/rh-redis5/